consensus/processor.go

```diff
@@ -5,9 +5,13 @@ import (
 	"math/big"
 	"time"
 
+	"go.opencensus.io/stats"
```

Comment:
Hopefully we'll be on go modules soon, but until then, these need to be gx-ified.

Reply:
I can do that -- think they already exist somewhere as gx packages.